playwright/support/atk_commands.js

```diff
@@ -269,20 +259,39 @@ function getDrushAlias () {
   return cmd
 }
 
+/**
+ * Get user as an object by user e-mail
```

Comment:
We're using Drupal conventions for comments i.e. please use sentences that end with a period.

Please see:
https://www.drupal.org/docs/develop/standards